ansible_events_ui/schemas.py

```diff
@@ -77,6 +76,28 @@ class Activation(BaseModel):
     extra_var_id: int
 
 
+class ActivationRead(ActivationCreate):
+    id: int
+    activation_status: Optional[StrictStr]
+    restarted_at: Optional[datetime]
+    restarted_count: int
+    created_at: datetime
+    modified_at: datetime
+    rulebook_name: StrictStr
+    inventory_name: StrictStr
+    extra_var_name: StrictStr
+    playbook_name: StrictStr
+    restart_policy_name: StrictStr
+    execution_env_url: StrictStr
+
+
+class ActivationUpdate(BaseModel):
+    id: Optional[int]
```

Comment:
The id is taken from the path param, so I think it's not necessary here, but it's included in the response, isn't? Maybe you are confusing the request model with the response model?